lib/internal/Magento/Framework/Backup/AbstractBackup.php

```diff
@@ -296,4 +304,24 @@ protected function _filterName($name)
 
         return $name;
     }
+
+    /**
+     * Set if keep files of backup
+     *
+     * @param bool $keepSourceFile
+     * @return $this
+     */
+    public function setKeepSourceFile(bool $keepSourceFile)
```

Comment:
So as this class has been marked with the `@api` notation we cannot accept new methods here.